lib/pacemaker-service.pc.in

```diff
@@ -14,4 +14,4 @@ Requires.private: libxml-2.0 @PC_NAME_DBUS@
 Conflicts:
 Cflags:           -I${includedir}
 Libs:             -L${libdir} -l${sub}
-Libs.private:     @PC_LIBS_RT@
+Libs.private:     @RT_LIBS@
```

Comment:
And then here we're saying we need these libraries, but we never link against them in lib/services/Makefile.am :/

Obviously we wouldn't have been able to build anywhere that actually needed librt, so maybe we should just use REQUIRE_FUNC() after all and not look for a library